packages/suite/src/components/suite/SecurityCheck/SecurityCheckFail.tsx

```diff
@@ -50,30 +48,29 @@ export const SecurityCheckFail = ({ goBack, useSoftMessaging }: SecurityCheckFai
         ? 'TR_DEVICE_COMPROMISED_TEXT_SOFT'
         : 'TR_DEVICE_COMPROMISED_TEXT';
 
-    const { elevation } = useElevation();
-
     return (
```

Comment:
I prioritized it in our design backlog that we are going to polish these screens in near future.